shared/ui-components/styles/navbar.scss

```diff
@@ -3,10 +3,15 @@
 @import "~bulma/sass/components/navbar.sass";
 
 .navbar-notice {
+  @apply hidden;
```

Comment:
@slowbot Just checking…did we decide that the feedback notice should be hidden on mobile now?

Reply:
@jaredcwhite I think yes, @slowbot mentioned about this in #76 . @slowbot could you confirm?

Reply:
I re-read #76, and I agree with this assessment.